### PR TITLE
Dijkstra Algorithm [Go] #577

### PR DESCRIPTION
**Pull request by ErangaHeshan**
Fixes #{ISSUE_NUMBER}

#### By submitting this pull request I confirm I've read and complied with the below declarations.

- [x] I have read the [Contribution guidelines](https://github.com/iiitv/algos/blob/master/CONTRIBUTING.md) and I am confident that my PR reflects them.
- [x] I have followed the [coding guidelines](https://github.com/iiitv/algos/blob/master/CONTRIBUTING.md#cs) for this project.
- [x] My code follows the [skeleton code structure](https://github.com/iiitv/algos/blob/master/CONTRIBUTING.md#sample).
- [x] This pull request has a descriptive title. For example, `Added {Algorithm/DS name} [{Language}]`, not `Update README.md` or `Added new code`.
- [x] This pull request will be closed if I fail to update it even once in a continuous time span of 7 days.

### CODE REVIEW

**Comment by singhpratyush** [comment]
Hi! Thanks for showing interest in contributing to this repository.

Please make sure that you have ticked the points mentioned in PR description correctly.

Thanks again!

**Comment by mohitkyadav** [comment]
@ErangaHeshan Please make sure that you are assigned to the issue before sending a PR.

File: dijkstra/dijkstra.go [hunk 1]
```diff
@@ -0,0 +1,124 @@
...
+}
+
+func main() {
+
+	g := &graph{}
+	g.edges = make(map[int][]*edge)
+	g.nodes = make(map[int]struct{})
+
+	log.Println("Loading graph...")
+	g.load("dijkstraData.txt")
```
Comment (Monal5031) [inline dijkstra/dijkstra.go]: Hardcode data, in this file itself.
Take only 10-20 numbers for testing.

Comment (Monal5031) [inline dijkstra/dijkstra.go]: Hardcode data, in this file itself.
Take only 10-20 numbers for testing.

File: dijkstra/dijkstra.go [hunk 2]
```diff
@@ -0,0 +1,124 @@
+import (
```
Comment (mohitkyadav) [inline dijkstra/dijkstra.go]: Add `package main`.

**Pull request closed without merging**